provisioning_templates/provision/kickstart_default.erb

```diff
@@ -31,6 +31,7 @@ This template accepts the following parameters:
   rhel_compatible = @host.operatingsystem.family == 'Redhat' && @host.operatingsystem.name != 'Fedora'
   is_fedora = @host.operatingsystem.name == 'Fedora'
   os_major = @host.operatingsystem.major.to_i
+  os_minor = @host.operatingsystem.minor.to_i
```

Comment:
`minor` isn't mandatory, so could be `nil`? But `nil.to_i == 0`, so that should be fine.